test_framework/Dockerfile.setup

```diff
@@ -2,7 +2,7 @@ From alpine:latest
 
 ARG KUBECTL_VERSION=v1.20.2
 
-ARG RKE_VERSION=v1.2.9
+ARG RKE_VERSION=v1.3.4
```

Comment:
RKE v1.3.4 does include `v1.22.5-rancher1-1` K8s version, what's the default one as the v1.1.x will not work on v1.22.5?

Reply:
RKE v1.3.4 include the following k8s versions:
![image](https://user-images.githubusercontent.com/18232258/151458456-21288945-bc1e-48e8-914f-1367acd349fd.png)

However, I've fixed the default to be `v1.21.8-rancher1-1`